omero/sysadmins/public.rst

```diff
@@ -111,6 +111,12 @@ To set this up on your OMERO.web installation:
 
      $ bin/omero config set omero.web.public.server_id 1
 
+- By default the public user is only allowed to perform GET requests. You can 
```

Comment:
Maybe add a little more info. 2nd sentence:
"This means that the public user will not be able to Create, Edit or Delete data which requires POST requests."

Also, since this config affects the ```url_filter``` setting, maybe this section should be moved up, above the ```url_filter``` section.